Moli/Core/Screens/Profile/ProfileView.swift

```diff
@@ -209,116 +221,104 @@ struct ProfileView: View {
                         UserThumbnailView(userMinified: profileUserImage(), size: .max)
                     }
                     .overlay(Circle().strokeBorder(Color.primaryDark, lineWidth: 3, antialiased: true))
-                    .offset(x: profileImageSizeXOffset, y: profileImageSizeYOffset)
-                }
+                    .offset(x: 0, y: profileImageSizeYOffset)
+                }.padding(.bottom, 40)
                 
-                VStack(alignment: .leading, spacing: 12) {
-                    HStack(spacing: 4) {
-                        Spacer()
-                        
-                        EditProfileView()
-                            .navigatePush(when: $isShowingEditProfile)
-                        
-                        AsyncButton(action: onPressFollowButton) {
-                            Text(followButtonText())
-                        }
-                        .buttonStyle(SuperButtonStyle(.roundedOutline(type: isFollowing() ? .secondary : .primary), size: .small))
-                    }
-                    
-                    VStack(alignment: .leading) {
+                VStack(alignment: .center, spacing: 16) {
```

Comment:
:art: Här skulle det räcka med att förkorta eftersom by default så har en VStack alignment: .center, och spacing: 16
```suggestion
                VStack {
```